opal-core-ws/src/main/java/org/obiba/opal/web/vcf/VCFStoreResourceImpl.java

```diff
@@ -90,6 +95,20 @@ public Response deleteSamplesMapping() {
     return Response.noContent().build();
   }
 
+  @Override
+  public Response getStatistics(String vcfName) {
+    StreamingOutput stream = new StreamingOutput() {
+      @Override
+      public void write(OutputStream os) throws IOException {
+        store.readVCFStatistics(vcfName, os);
+        new BufferedWriter(new OutputStreamWriter(os)).flush();
```

Comment:
What's the use of the BufferedWriter? at this point the data are alreay written in 'os'